bindings/python/pinocchio/visualize/panda3d_visualizer.py

```diff
@@ -121,3 +117,5 @@ def displayVisuals(self, visibility):
         """Set whether to display visual objects or not."""
         self.viewer.show_group(self.visual_group, visibility)
         self.display_visuals = visibility
+        
+__all__ = ['Panda3dVisualizer']
```

Comment:
@jcarpent removing `Panda3dViewer` from `__all__` breaks the examples written in Pinocchio.

I am not familiar with the code, but it seems you need to put back the previous `__all___` definition.

Reply:
@cmastalli @ikalevatykh The changes are relevant, as panda3d_viewer may not be available. Then, you cannot import the lib, otherwise, you will get an error.

Reply:
@jcarpent Original variant was also OK as panda3d_visualize was not imported in visualize. _import_. Just to clarify